server/chat-plugins/rock-paper-scissors.ts

```diff
@@ -0,0 +1,404 @@
+/**
+ * Rock, Paper, Scissors plugin by Mia
+ * @author mia-pi-git
+ */
+import {Utils} from '../../lib/utils';
+
+const TIMEOUT = 10 * 1000;
+const MAX_ROUNDS = 10;
+const ICONS: {[k: string]: string} = {
+	Rock: `<i class="fa fa-circle"></i>`,
+	Paper: '<i class="fa fa-file"></i>',
+	Scissors: '<i class="fa fa-scissors"></i>',
+};
```

Comment:
Umm... did you look at how the teambuilder implements these. :p

Reply:
I did not, i'll do so now.